fuel-vm/src/tests/alu.rs

```diff
@@ -71,35 +71,20 @@ fn reserved_register() {
 }
 
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
-pub enum AluResult {
-    Success { value: Word, of: Word, err: Word },
-    MissingLog,
-    Revert,
-    Panic(PanicReason),
-    GenericFailure(u64),
+pub struct AluOk {
```

Comment:
Do we need to touch it in this PR?=)

Reply:
The `RunResult` changes must be done here, and these are a natural consequence of those. It would be weird not to do them here.